sqlparse/sql.py

```diff
@@ -251,7 +216,7 @@ def _token_matching(self, funcs, start=0, end=None, reverse=False):
                 if func(token):
                     return token
 
-    def token_first(self, ignore_whitespace=True, ignore_comments=False):
+    def token_first(self, skip_ws=True, skip_cm=False):
```

Comment:
I want to say there were certain functions that were called `skip_ws` and others called `ignore_whitespace` and was cleaning that up. If I recall they would be called any of `ignore_whitespace` `ignore_ws` `skip_ws`. I'll double check to see if i find an example of it.